src/actions/signup.actions.js

```diff
@@ -0,0 +1,14 @@
+import api from '../api';
+import SIGNUP from '../constants/signup.constants';
+
+// Action creator
+const signUpAct = credentials => ({
```

Comment:
Rename this to something more descriptive. Either leave it as `signup` or call it `signupUser`. We already know its an action since the file name tells as much so we don't need to call it `signupAct `

Reply:
Done